site/en/install/pip.md

```diff
@@ -75,8 +76,9 @@ step-by-step instructions.
     ```bash
     conda install -c conda-forge cudatoolkit=11.8.0
     python3 -m pip install nvidia-cudnn-cu11==8.6.0.163 tensorflow==2.12.*
-    CUDNN_PATH=$(dirname $(python -c "import nvidia.cudnn;print(nvidia.cudnn.__file__)"))
-    export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$CONDA_PREFIX/lib/:$CUDNN_PATH/lib
+    mkdir -p $CONDA_PREFIX/etc/conda/activate.d
+    echo 'CUDNN_PATH=$(dirname $(python -c "import nvidia.cudnn;print(nvidia.cudnn.__file__)"))' > $CONDA_PREFIX/etc/conda/activate.d/env_vars.sh
+    echo 'export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$CONDA_PREFIX/lib/:$CUDNN_PATH/lib' >> $CONDA_PREFIX/etc/conda/activate.d/env_vars.sh
```

Comment:
same here.

Reply:
Thank you for pointing it out. I've made the suggested changes.